README.md

```diff
@@ -42,6 +42,104 @@ This will construct the platform specific build environment. For example, on
 Windows with a properly configured Visual Studio, this will create in the
 directory `build.d` the solution file `Resonance.sln`.
 
+# Example
+
+The following example is adapted from the `RunMatrixMultiply` test of the ONNX

```

Comment:
Is it possible to link to the onnx repo so that someone can refer to `RunMatrixMultiply` easily? Is it this one that you are referring to? https://github.com/onnx/onnx/blob/master/docs/Operators.md#MatMul

Reply:
Well, I meant our test `RunMatrixMultiply`. I'll update the description to make it more clear. But certainly a link to the ONNX documentation on what the operator is that is underlying the model we're playing with certainly couldn't hurt for sure!